### PR TITLE
Update 'TFX demo link'

### PR DESCRIPTION
**Pull request by chensun**
Link to GitHub code rather than the notebook opening page

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3150)

### CODE REVIEW

File: frontend/src/pages/__snapshots__/GettingStarted.test.tsx.snap [hunk 1]
```diff
@@ -73,14 +73,14 @@ exports[`GettingStarted page initially renders documentation 1`] = `
            
           <ul>
             <li>
-              Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. Learn how to 
+              Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. 
               <a
                 class="link"
-                href="https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252Fmaster%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb"
+                href="https://github.com/kubeflow/pipelines/tree/master/samples/core/tfx-oss"
```
Comment (numerology) [inline frontend/src/pages/__snapshots__/GettingStarted.test.tsx.snap]: ditto

File: frontend/src/pages/GettingStarted.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ This section contains demo and tutorial pipelines.
 
 **Demos** - Try an end-to-end demonstration pipeline.
 
-  * [TFX pipeline demo](${tfx}) - Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. Learn how to [get started with TFX pipeline!](https://console.cloud.google.com/mlengine/notebooks/deploy-notebook?q=download_url%3Dhttps%253A%252F%252Fraw.githubusercontent.com%252Fkubeflow%252Fpipelines%252Fmaster%252Fsamples%252Fcore%252Fparameterized_tfx_oss%252Ftaxi_pipeline_notebook.ipynb)
+  * [TFX pipeline demo](${tfx}) - Classification pipeline with model analysis, based on a public BigQuery dataset of taxicab trips. [source code](https://github.com/kubeflow/pipelines/tree/master/samples/core/tfx-oss)
```
[the remark this answers is not in the document]
Comment (chensun) [reply frontend/src/pages/GettingStarted.tsx]: Sync'ed offline. Will use https://github.com/kubeflow/pipelines/tree/master/samples/core/parameterized_tfx_oss

**Pull request merged**